CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # ChangeLog
 
-## **next**
+## **5.82** (2023-04-28)
```

Comment:
5.82.0

I believe this is what we usually.

Reply:
It cannot be changed once the blog post and emails are published because it would break the link.

Reply:
#6805